python/dask_cudf/dask_cudf/io/json.py

```diff
@@ -49,7 +49,7 @@ def read_json(url_path, engine="auto", **kwargs):
 
     Load nested JSON data
 
-    >>> read_json('myfile.json', engine='cudf_experimental')  # doctest: +SKIP
+    >>> read_json('myfile.json', engine='cudf')  # doctest: +SKIP
```

Comment:
```suggestion
    >>> read_json('myfile.json')  # doctest: +SKIP
```

Reply:
I think we want this to be `cudf` instead of `auto`, for the same reason as: https://github.com/rapidsai/cudf/pull/12509/files#r1083132742

Reply:
Yup, this actually worked. Checked locally and added a test for this.